datasource/src/main/scala/quasar/physical/mongo/MongoDatasource.scala

```diff
@@ -33,13 +34,12 @@ import quasar.physical.mongo.MongoResource.{Collection, Database}
 import quasar.qscript.InterpretedRead
 
 final class MongoDataSource[F[_]: ConcurrentEffect: ContextShift: MonadResourceErr: Timer](
-  mongo: Mongo[F]) extends LightweightDatasource[Resource[F, ?], Stream[F, ?], QueryResult[F]] {
+    mongo: Mongo[F])
+    extends LightweightDatasource[Resource[F, ?], Stream[F, ?], QueryResult[F]] {
 
   val kind = MongoDataSource.kind
 
-  val loaders = NonEmptyList.of(
-    Loader.Batch(BatchLoader.Full(loader(_, none))),
-    Loader.Batch(BatchLoader.Seek(loader(_, _))))
+  val loaders = NonEmptyList.of(Loader.Batch(BatchLoader.Seek(loader(_, _))))
```

Comment:
A `Seek` loader may always be used to service full loads by omitting the offset. In the case where there is a material difference in implementation of both types for a particular datasource, we could conceivably implement both loaders, but in this case having both is unnecessary.